FrontEnd/src/components/SignUp/components/signup/signup-form/SignUpFormContent.js

```diff
@@ -67,7 +67,11 @@ export function SignUpFormContentComponent(props) {
       url: `${process.env.REACT_APP_BASE_API_URL}/api/auth/users/`,
       data: dataToSend,
     })
-      .then()
+      .then((res) => {
+        if (res.status === 201) {
```

Comment:
Do we receive another success status? If no we can remove the if

Reply:
Done